opencti-platform/opencti-front/src/private/components/workspaces/investigations/InvestigationGraph.jsx

```diff
@@ -954,6 +972,9 @@ class InvestigationGraphComponent extends Component {
       params,
     );
     const timeRangeInterval = computeTimeRangeInterval(this.graphObjects);
+
+    this.fetchMetaObjectRelCounts(this.graphObjects);
```

Comment:
We fetch asynchronously the number of relationships of meta objects to avoid waiting too long to display the graph as the query can take some times.
We display a placeholder in the graph until the query is resolved.